vhdbuilder/packer/test/vhd-content-test.ps1

```diff
@@ -0,0 +1,189 @@
+<#
+    .SYNOPSIS
+        verify the content of Windows image built
+    .DESCRIPTION
+        This script is used to verify the content of Windows image built
+#>
+
+param (
+    $containerRuntime
+)
+
+# TODO(qinhao): we can share the variables from configure-windows-vhd.ps1
+$global:containerdPackageUrl = "https://marosset.blob.core.windows.net/pub/containerd/containerd-0.0.87-public.zip"
```

Comment:
The latest package is `https://kubernetesartifacts.azureedge.net/containerd/ms/0.0.11-1/binaries/containerd-windows-0.0.11-1.zip` now.